html/changelog.html

```diff
@@ -59,6 +59,8 @@ <h2 class="date">31 May 2015</h2>
 			<h3 class="author">duncathan updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="tweak">Clarified the ability to retract changeling armblades.</li>
+                <li class="soundadd">Wizard Spells and Wands now have unique sounds, made by Cuboos</li>

```

Comment:
I fixed it, sorta, i did the right method but now how do i remove the HTML file?